stacks/core/dev/stage-1.yml

```
@@ -90,6 +105,15 @@ services:
     volumes:
       - mainflux-authn-db-volume:/var/lib/postgresql/data
 
+  mongo:
+    image: mongo
+    volumes:
+      - mongo-data:/data/db
```

Could we modify this so we are using a standard for `volumes`? Something like this :point_down: 
```suggestion
      - knot-cloud-storage-db-volume:/var/lib/mongo/data
```
>I suggested the `knot-cloud-storage` prefix, instead of Babeltower, considering the storage is transparent to Babeltower. Tho, It could also be :point_down: 
```suggestion
      - knot-cloud-db-volume:/var/lib/mongo/data
```

---

Done! Accepted second suggestion :smile: